code_benches/run.py

```diff
@@ -27,32 +44,59 @@
 
 RUNTIME_PATH = ROOT_PATH + "/runtime"
 
-SILVERFISH_RELEASE_PATH = ROOT_PATH + "/target/release/silverfish"
-SILVERFISH_DEBUG_PATH = ROOT_PATH + "/target/debug/silverfish"
-assert all(arg in {"--release", "--debug"} for arg in sys.argv[1:])
-if "--release" in sys.argv:
-    SILVERFISH_PATH = SILVERFISH_RELEASE_PATH
-elif "--debug" in sys.argv:
-    SILVERFISH_PATH = SILVERFISH_DEBUG_PATH
-else:
-    def getmtime_or_zero(path):
+def bestpath(paths):
+    """
+    Determine best path based on:
+    1. exists
+    2. most recently modified
+    """
+    def getmtime_or_zero(x):
+        path, *_, use_this_one = x
+
+        if use_this_one:
+            return (1, 0)
+
         try:
-            return os.path.getmtime(path)
+            return (0, os.path.getmtime(path))
         except FileNotFoundError:
-            return 0
-    SILVERFISH_PATH = max(
-        [SILVERFISH_RELEASE_PATH, SILVERFISH_DEBUG_PATH],
-        key=getmtime_or_zero)
+            return (0, 0)
+
+    *best, _ = max(paths, key=getmtime_or_zero)
+    return best
+
+SILVERFISH_RELEASE_PATH = ROOT_PATH + "/target/release/silverfish"
+SILVERFISH_DEBUG_PATH = ROOT_PATH + "/target/debug/silverfish"
+
+SILVERFISH_PATH, = bestpath([
+    (SILVERFISH_RELEASE_PATH, args.release),
+    (SILVERFISH_DEBUG_PATH,   args.debug),
+])
```

Comment:
What happens if someone passes in both `--release` and `--debug`?

Reply:
Dunno. Probably breaks.

I didn't think it was worth enforcing, but I can look into how to enforce exclusivity if you think it is worth it.

Reply:
Do you know of an argparse option that would help?

Reply:
Still don't want to merge with out an explicit check for this edge case. 

Also can you clean up the name and documentation of the "best path" function? It's a little more specific than the name let's on

Reply:
I think this documentation should help with this: https://docs.python.org/3/library/argparse.html#mutual-exclusion